src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -184,6 +186,55 @@ class ChatHighlightBlacklistKeywordsModule {
             defaultValue: false,
             description: 'Automatically hide pinned highlights after 1 minute'
         });
+
+        settings.add({
+            id: 'soundHighlight',
+            name: 'Play Sound On Highlight',
+            defaultValue: false,
+            description: 'On highlight you hear a sound'
+        });
+
+        this.username = twitch.getChatServiceClient().configuration.username.toLowerCase();
+        this.soundHighlight = false;
+
+        this.setSoundHighlight = this.setSoundHighlight.bind(this);
+        this.handleSoundHighlight = this.handleSoundHighlight.bind(this);
+
+        this.setSoundHighlight(settings.get('soundHighlight'));
+        settings.on('changed.soundHighlight', this.setSoundHighlight);
+    }
+
+    setSoundHighlight(value) {
+        this.soundHighlight = value;
+    }
+
+    handleSoundHighlight(messageParts = []) {
+        const isMentioned = messageParts.some(({content}) => {
```

Comment:
why don't we just play the sound from at line 271 and skip all this "mentioned" logic?

Reply:
So scrap the mentioned logic and add this for highlights instead?